adapters/Pangle/PangleAdapter/GADMediationAdapterPangle.m

```diff
@@ -63,22 +108,45 @@ + (GADVersionNumber)adapterVersion {
 
 - (void)loadBannerForAdConfiguration:(GADMediationBannerAdConfiguration *)adConfiguration
                    completionHandler:(GADMediationBannerLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding banner ad.
+    [BUAdSDKManager setCoppa:adConfiguration.childDirectedTreatment.integerValue];
+    _bannerRenderer = [[GADPangleRTBBannerRenderer alloc]init];
```

Comment:
Nit, spacing: `_bannerRenderer = [[GADPangleRTBBannerRenderer alloc] init];`

The same applies to all the other places.